### PR TITLE
Adds me to larva

### LINKED ISSUE #12147: Xeno larvas don't have me emotes.
While #11358 added me emotes for xenos and monkeys, the xeno larvas still don't have it. I'm gonna assume this an oversight.

### PR DESCRIPTION
**Pull request by ghost**
Adds a unique *deathgasp message and the ability to do me emotes when as a xeno larva.

**Larva 969** curls up into a bundle as they watch the spessmen bicker and fight.

Fixes #12147 (or rather implements the request but I hear Fixes X is how it auto closes the issues?)
@WJohn, enjoy.

### CODE REVIEW

**Comment by WJohn** [comment]
:+1: 
Thank you <3

**Comment by Bawhoppen** [comment]
There never was a good reason they didn't have emotes originally.

**Comment by RemieRichards** [comment]
@CosmicScientist 

Github accepts:
close
closes
closed
fix
fixes
fixed
resolve
resolves
resolved

**Comment by CosmicScientist** [comment]
@RemieRichards thank you.

File: code/modules/mob/living/carbon/alien/larva/emote.dm [hunk 1]
```diff
@@ -95,10 +99,10 @@
 				m_type = 2
 
 		if ("help") //"The exception"
-			src << "Help for larva emotes. You can use these emotes with say \"*emote\":\n\nburp, choke, collapse, dance, drool, gasp, gnarl, hiss, jump, moan, nod, roll, roar, scratch, screech, shake, shiver, sign-#, sulk, sway, tail, twitch, whimper"
+			src << "Help for larva emotes. You can use these emotes with say \"*emote\":\n\nburp, choke, collapse, dance, deathgasp, drool, gasp, gnarl, hiss, jump, me, moan, nod, roll, roar, scratch, screech, shake, shiver, sign-#, sulk, sway, tail, twitch, whimper"
 
 		else
-			src << "<span class='info'>Unusable emote '[act]'. Say *help for a list.</span>"
+			..(act)
```
Comment (phil235) [inline code/modules/mob/living/carbon/alien/larva/emote.dm]: pretty sure this is going to give larvas a whole lot of emotes that they shouldn't have.

Comment (CosmicScientist) [inline code/modules/mob/living/carbon/alien/larva/emote.dm]: Oh I'll take that out then, sorry my bad.

**Pull request merged**